sympy/tensor/array/array_comprehension.py

```diff
@@ -48,33 +47,44 @@ def __new__(cls, function, *symbols, **assumptions):
 
     @property
     def function(self):
-        """
-        Return the function applied across limits
+        """Use for obtaining the function applied across limits
+
+        Returns
+        =======
+
+        Expr
+            The applied function
 
         Examples
         ========
 
         >>> from sympy.tensor.array import ArrayComprehension
-        >>> from sympy.abc import i, j, k
+        >>> from sympy import symbols
+        >>> i, j = symbols('i j')
         >>> a = ArrayComprehension(10*i + j, (i, 1, 4), (j, 1, 3))
         >>> a.function
         10*i + j
         """
-        return self._function
+        return self._args[0]
 
     @property
     def limits(self):
         """
-        Return a list of the limits that will be applied while expanding the array. Each
-        bound contrains firstly the an variable (not necessarily to be component of the
-        expression, e.g. an array of constant). Then the lower bound and the upper bound
-        define the length of this expansion.
+        Used for obtaining the list of limits that will be applied
```

Comment:
`Used for ...` unnecessary words, just write straight what it does: the list of limits.